api/lib/mno_enterprise/concerns/controllers/jpi/v1/marketplace_controller.rb

```diff
@@ -31,14 +30,14 @@ def index
 
   # GET /mnoe/jpi/v1/marketplace/1
   def show
-    @app = MnoEnterprise::App.find(params[:id])
+    @app = MnoEnterprise::App.find_one(params[:id])
   end
 
   def app_relation
     if MnoEnterprise.marketplace_listing
-      MnoEnterprise::App.where('nid.in' => MnoEnterprise.marketplace_listing)
+      MnoEnterprise::App.where(nid: MnoEnterprise.marketplace_listing)
     else
-      MnoEnterprise::App.all
+      MnoEnterprise::App.where
```

Comment:
I confirm,  there is ` alias all to_a` in the builder, and here I need a query builder.